example/App.tsx

```diff
@@ -0,0 +1,250 @@
+import React, {useCallback, useEffect, useState} from 'react';
```

Comment:
Is it correct this file is marked as added? I know you made changes to the example in another PR

Reply:
Mistake, removing. Nice catch.